src/cmd-cloud-prune

```diff
@@ -320,5 +350,57 @@ def delete_gcp_image(build, cloud_config, dry_run):
     return errors
 
 
+def prune_images(s3, build, images_to_keep, dry_run, bucket, prefix):
+    images_from_meta_json = build.meta_json.get("images", [])
+    # Get the image names and paths currently in meta.json
+    current_images_data = [(name, data.get("path")) for name, data in images_from_meta_json.items()]
+    # Extract names to compare with images_to_keep
+    current_images = [name for name, _ in current_images_data]
+    errors = []
+
+    if images_to_keep == current_images:
```

Comment:
can we compare lists like this? does order matter?

Reply:
noob mistake. The order does matter